src/Stratis.Bitcoin.IntegrationTests/ConsensusManagerTests.cs

```diff
@@ -265,8 +265,8 @@ public void ConsensusManager_Reorgs_Then_Old_Chain_Becomes_Longer_Then_Reorg_Bac
 
                 // Miner A mines to height 25.
                 TestHelper.MineBlocks(minerA, 10);
-                TestHelper.WaitLoop(() => TestHelper.AreNodesSynced(syncer, minerA));
-                TestHelper.WaitLoop(() => TestHelper.AreNodesSynced(syncer, minerB));
+                TestHelper.WaitLoopMessage(() => TestHelper.AreNodesSyncedMessage(syncer, minerA), waitTimeSeconds: 120);
```

Comment:
this is odd why would the rate limiter require longer wait time?

Reply:
Ahh sorry, this was just for debugging the tests. I can remove this actually?